core/templates/dev/head/pages/creator_dashboard/create_question_modal_directive.html

```diff
@@ -0,0 +1,15 @@
+<div class="modal-header">
+  <h3>Select topic to create question in</h3>
+  <select ng-model="topicId" ng-options="topicSummary.id as topicSummary.name for topicSummary in topicSummaries"></select>
+  <h3>Create Question</h3>
+</div>
+
+<div class="modal-body">
+  <question-editor question-id="null" question-state-data="questionStateData" misconceptions="[]" can-edit-question="true">
```

Comment:
The previous comment I made on this was not addressed, although it was marked "Done". Don't pass lists directly as attr values.

Reply:
Done